bioscrape/inference.pyx

```diff
@@ -256,18 +321,28 @@ cdef class ModelLikelihood(Likelihood):
             for (key,val) in  pairs:
                 self.init_state_indices[index] = self.m.get_species_index( key  )
                 self.init_state_vals[index] = val
-                index +=  1
+                index +=  1"""
```

Comment:
Should these """ be here?

Reply:
yeah, it's the multiline comment end. git diff detected it incorrectly. I put these """ in a new line so it is clearer.

Reply:
Great. Otherwise Looks good.